boilerplate/openshift/custom-catalog-osd-operator/custom-catalog-build-push.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+set -ev
+
+# Global vars
+CONTAINER_ENGINE=$(command -v podman || command -v docker)
+CONTAINER_ENGINE_SHORT=${CONTAINER_ENGINE##*/}
+REPO_ROOT=$(git rev-parse --show-toplevel)
+VERSIONS_DIR=${REPO_ROOT}/versions
+
+source $REPO_ROOT/boilerplate/_lib/common.sh
+
+function usage() {
+    cat <<EOF
+    Usage: $0 REGISTRY_IMAGE_URI
+    REGISTRY_IMAGE_URI is the complete image URI that needs to be built
+EOF
+    exit -1
+}
+
+function build_catalog_image() {
+  local bundle_image=${1}
+  local image_tag=${2}
+  local opm_local_executable
+
+  # install opm binary if it does not exist
+  if which opm >/dev/null 2>&1; then
+      opm_local_executable=$(realpath $(which opm))
```

Comment:
Note that this will use whatever version of opm the user happens to have installed, which may not behave the way you expect. Might be better to omit this branch entirely, and force use of `install-opm`.

Reply:
ok will do, thanks